src/app/Dashboard/components/tests/LogPointsModal.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { LogActivityForm } from '../LogPointsModal';
+
+describe('<LogPointsComponent />', () => {
+  const setUpWrapper = ({
+    categories = [],
+  } = {}) => {
+    const props = {
+      categories,
+      loadCategories: jest.fn(),
+      handleChange: jest.fn(),
+    };
+    const shallowWrapper = shallow(<LogActivityForm {...props} />);
+    return {
+      shallowWrapper,
+    };
+  };
+
+  it('should submit the log society activity points data', () => {
```

Comment:
You missed the `expect block` in this and test case. Also, the one that follows.

Reply:
@Chris-Maina Thank you for catching that. I have added the except block